TravelerApp/TravelerAppCore/Controller/ReadWriteHotel.cs

```diff
@@ -0,0 +1,124 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using TravelerAppConsole;
+using TravelerAppCore.Models.Hotels;
+using TravelerAppCore.View;
+
+namespace TravelerAppCore.Controller
+{
+    public class ReadWriteHotel
+    {
+        public static void ReadAndDisplay(List<Hotel> dataReaded)
+        {
+            Read(dataReaded);
+            DisplayLoadedData(dataReaded);
+        }
+        public static Stopwatch stopper = new Stopwatch();
+        public static void Read(List<Hotel> dataReaded)
+        {
+            stopper.Start();
+            dataReaded.Clear();
+            string sCurrentDirectory = AppDomain.CurrentDomain.BaseDirectory;
```

Comment:
co to za 's' przy nazwach?

Reply:
Notacja węgierska. Sposób zapisu nazw zmiennych oraz obiektów, polegający na poprzedzaniu właściwej nazwy małą literą (literami) określającą rodzaj tej zmiennej (obiektu). Stąd s od string. Nie umiałem się zdecydować na nazwę zmiennej stąd nie wszędzie tego stosuję. Do poprawienia w każdym razie

Reply:
domyśliłem się że to od typu bardziej chodziło mi o to w jakim celu? przecież widać ze to string :) przy okazji notacja węgierska nie jest zalecana :)

Reply:
Poprawiono